api/v1/routes/superadmin.py

```diff
@@ -0,0 +1,61 @@
+"""
+Superadmin endpoints
+"""
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from typing import Annotated
+from sqlalchemy.orm import Session
+
+# from fastapi.responses import JSONResponse
+from api.utils.success_response import success_response
+from api.db.database import get_db
+from api.v1.models.user import User
+from api.v1.services.user import user_service
+from uuid import UUID
+
+superadmin = APIRouter(prefix="/superadmin", tags=["superadmin"])
+
+
+@superadmin.delete("/users/{user_id}")
+def delete_user(
+    user_id: UUID,
+    current_user: Annotated[User, Depends(user_service.get_current_user)],
```

Comment:
There is a get_current_super_admin in the user service. This saves you the stress of checking in the routes again

Reply:
resolve this please

Reply:
resolved